packages/common-substrate/src/project/project.spec.ts

```diff
@@ -28,6 +28,24 @@ describe('project.yaml', () => {
     expect(() => loadSubstrateProjectManifest(path.join(projectsDir, 'project_0.2.0_custom_ds.yaml'))).not.toThrow();
   });
 
+  it('can validate a v0.2.1 project.yaml with templates', () => {
```

Comment:
What do you mean `with templates`. Can you add tests for terra manifest as a seperated one?

Reply:
As manifest 0.2.1 is for templates support, it should be able to validate a manifest with templates